recipe/meta.yaml

```diff
@@ -13,17 +13,19 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 0
+  number: 1
+  noarch: python
   entry_points:
     - pybtex = pybtex.__main__:main
     - pybtex-convert = pybtex.database.convert.__main__:main
     - pybtex-format = pybtex.database.format.__main__:main
-  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Comment:
You'll want to remove the `--no-cache-dir` option here, see https://github.com/conda/conda-build/issues/3094.
I'm not sure how to *really* make sure to get `wheel`'s `.dist-info` instead of a `-py?.?.egg-info`, though :/